lldb/include/lldb/Utility/ProcessInfo.h

```diff
@@ -237,6 +250,16 @@ class ProcessInstanceInfo : public ProcessInfo {
            m_cumulative_system_time.tv_usec > 0;
   }
 
+  int8_t GetNiceValue() const { return m_nice_value; }
```

Comment:
Suggestion: `nice` -> `priority_level` or something similar. The unix world uses the words `nice` and `niceness` but Windows and other platforms do not. It's also not a very obvious word if you lack the domain expertise.

Reply:
nice and priority are not synonymous though, nice is one input into the process's priority. I'm not sure which we actually want in this context.

Reply:
Done. Agreed. I also added a note in the linux specific code. In linuxland priority implies a realtime priority while nice is nice

Reply:
prpsinfo has pr_nice field to be populated https://github.com/llvm/llvm-project/blob/main/lldb/source/Plugins/Process/elf-core/ThreadElfCore.h#L99